### ISSUE TITLE
Can't use node-stress-single-test with Windows due to three minute timeout

### ISSUE THREAD

**Issue by Trott** [nodejs/build#1281]
I tried to run Windows stress tests for https://github.com/nodejs/node/issues/20836#issuecomment-391390203 to see how often it fails but after compiling for a long time, the builds hit a three-minute timeout  and don't run the tests. See https://ci.nodejs.org/job/node-stress-single-test/1858/nodes=win2016-1p-vs2017/console and https://ci.nodejs.org/job/node-stress-single-test/1857/nodes=win2016-1p-vs2017/console. 

```console
19:31:05      Creating library c:\workspace\node-stress-single-test\nodes\win2016-1p-vs2017\Release\mksnapshot.lib and object c:\workspace\node-stress-single-test\nodes\win2016-1p-vs2017\Release\mksnapshot.exp
19:31:05   Generating code
19:34:05 Build timed out (after 3 minutes). Marking the build as aborted.
19:34:05 Build was aborted
19:34:05 Notifying upstream projects of job completion
19:34:05 Finished: ABORTED
```

@nodejs/build

**Comment by joaocgreis** [comment]
This was added by @refack in https://ci.nodejs.org/job/node-stress-single-test/jobConfigHistory/showDiffFiles?timestamp1=2018-03-27_05-14-08&timestamp2=2018-04-07_18-17-38 (but might have been unintentional? Jenkins sometimes changes the jobs on the first save after plugins are updated).

Linking can take a very long time without output in our single processor machines. I've changed the time out from 3 minutes to 1 hour, we might need to decrease if this turns out to be too much.

Rebuilding jobs:
- 1857: https://ci.nodejs.org/job/node-stress-single-test/1859/
- Tried to rebuild 1858 as well but Jenkins is quite busy at the moment and it won't start.
